eu.html

```diff
@@ -170,13 +170,13 @@
 <p ><b><span>5.
 Who are we and how to contact us</span></b></p>
 
-<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;TransferWise
+<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;Wise
 Europe NV/SA is a company incorporated under the law of Belgium with company
-number 0713.629.988 (&quot;TransferWise&quot;, &quot;we&quot;, &quot;us&quot;,
+number 0713.629.988 (&quot;Wise&quot;, &quot;we&quot;, &quot;us&quot;,
 or &quot;our&quot; as applicable).</span></p>
 
 <p ><b><span >5.2 Our Belgian Registered office.</span></b><span>&nbsp;Our
-registered office is TransferWise Europe SA/NV, Avenue Louise 54, Room s52,
+registered office is Wise Europe SA/NV, Avenue Louise 54, Room s52,

```

Comment:
Should say this

5.2 Our Belgian Registered office. Our registered office is TransferWise Europe SA/NV, Avenue Louise 54, Room s52, 1050 Brussels, Belgium.